backend/pkg/rest/runtime_scan_controller.go

```diff
@@ -61,8 +61,6 @@ func (s *Server) PutRuntimeScanStart(params operations.PutRuntimeScanStartParams
 func (s *Server) PutRuntimeScanStop(_ operations.PutRuntimeScanStopParams) middleware.Responder {
 	// stop scanner
 	s.runtimeScanner.Clear()
-	// stop listening for scanner results
-	s.runtimeScanner.StopCurrentScan()
```

Comment:
why removed? how do we stop a scan now?

Reply:
The "Clear" function which is used as our stop function everywhere else is now used to stop the scan instead of splitting it across two functions like

Reply:
Did you check it? not sure how is the scanner notified upon stop. Since he is waiting only for done on startScan func

Reply:
Yeah, ran and aborted a scan from the UI and it works well. The "done" channel notification now indicates complete in any form, whether aborted, failed or completed, and the scan progress is used to determine what state the scan was in when it is "done".

Reply:
I see it happens via the killSignal